app/plugin.go

```diff
@@ -751,7 +736,7 @@ func (a *App) processPrepackagedPlugins(pluginsDir string) []*plugin.Prepackaged
 		return nil
 	}
 
-	fileStorePaths := []string{}
+	var fileStorePaths []string
```

Comment:
Made declaring variable consistent. Here is the reasoning: https://dave.cheney.net/2014/05/24/on-declaring-variables.